components/ResultsPageComponents/CommuteDistanceDistribution/CommuteDistanceDistributionChart.jsx

```diff
@@ -17,27 +19,39 @@ export default function CommuteDistanceDistributionChart({ title, data }) {
   const highChartsOptions = {
     title: {
       text: title,
+      style: { color: "#022640" },
     },
     chart: {
       type: "column",
+      height: 400,
     },
     colors: ["#044B7F"],
     tooltip: {
-      pointFormat: "{point.y}%",
+      pointFormat: "{point.y}",
```

Comment:
we want to add the '%' symbol back to this...
```suggestion
      pointFormat: "{point.y}%",
```